crates/polars-time/src/chunkedarray/rolling_window/dispatch.rs

```diff
@@ -130,15 +132,7 @@ where
         let ca = unsafe { ca.take_unchecked(&sorting_indices) };
         let by = unsafe { by.take_unchecked(&sorting_indices) };
         let arr = ca.downcast_iter().next().unwrap();
-        if arr.null_count() > 0 {
-            polars_bail!(InvalidOperation: "'Expr.rolling_*_by(...)' not yet supported for series with null values, consider using 'DataFrame.rolling' or 'Expr.rolling'")
-        }
-        let by_values = by.cont_slice().map_err(|_| {
-            polars_err!(
-                ComputeError:
-                "`by` column should not have null values in 'rolling by' expression"
-            )
-        })?;
+        let by_values = by.cont_slice().expect("checked already");
```

Comment:
same in L112 - I can do this in a separate PR if you like though, where I'll add tests to check that this raises on null values - it's a bit concerning that in the previous commit the tests passed